content/livesync/index.textile

```diff
@@ -1,48 +1,83 @@
-- "Subscribe":/livesync/models/models#subscribe := renders changes to the model state in your application in realtime.
+There are several known limitations to taken into consideration where LiveSync may not be a good fit for your application, these are:
 
-The Models SDK is built on "Ably's JavaScript SDK":https://github.com/ably/ably-js and can be easily integrated into any front-end framework. The key advantages include:
+- Postgres is the only datastore currently supported,
+- applications that use highly relational or graph structured database will not perform well,
+- lists that require pagination will not be stored in a manageable data model,
+- when changes to a single data model will need to be reflected in other data models, these other data models will need their own models,
+- there are no further updates except for those performed by one user, there is no need for a synchronous method between the database and the one frontend client,
+- applications where peers are trusted, data can be handled entirely peer-to-peer,
+- when functionality allows two users to collaborate on the same element in a collaborative form editing and building application. Conflict resolution will discard one of the collaborator's updates.
 
-* "Authentication":/auth and "connection management":/connect are efficiently managed by the underlying Ably JavaScript SDK.
+h2(#pricing). Pricing
 
-* The Ably JavaScript SDK enables you to extend your application's functionality with "Pub/Sub Channels":/products/channels.
+LiveSync "pricing":https://ably.com/pricing is based on the message consumption. This means that each update published from the Database Connector to Ably Channels is counted as a single message. The message then received by every frontend client subscribed to that channel is counted as one message. If, for example, one update is published by the Database Connector and there are three frontend clients subscribed, the one update will count as four messages and priced as such.
 
-h2(#dev-status). Development status
+h2(#development-status). Development status
 
-LiveSync is currently in the early release stage, primarily intended for experimental use and not yet suitable for production environments. Your "feedback":https://docs.google.com/forms/d/e/1FAIpQLSd00n1uxgXWPGvMjKwMVL1UDhFKMeh3bSrP52j9AfXifoU-Pg/viewform will help in prioritizing improvements and fixes in subsequent releases.
+LiveSync is in public alpha so that you can explore its capabilities. Your "feedback":https://docs.google.com/forms/d/e/1FAIpQLSd00n1uxgXWPGvMjKwMVL1UDhFKMeh3bSrP52j9AfXifoU-Pg/viewform will help prioritize improvements and fixes in a subsequent release. The features have been validated for a set of use-cases and are stable and perform well, but API changes may occur that impact functionality. The alpha is implemented to work under real-world situations and load, but you should use caution when deciding under what conditions to use it.
```

Comment:
```suggestion
LiveSync is in public alpha so that you can explore its capabilities. Your "feedback":https://docs.google.com/forms/d/e/1FAIpQLSd00n1uxgXWPGvMjKwMVL1UDhFKMeh3bSrP52j9AfXifoU-Pg/viewform will help prioritize improvements and fixes for subsequent releases. The features in this release have been validated for a set of use-cases and the alpha is implemented to work under real-world situations and load, but you should use caution when deciding under what conditions to use it.
```